SU-SWS/stanford_profile_helper
# D8CORE-3516: fixing the missing arrow icon

# READY FOR REVIEW

# Summary
- Fixing where webpack puts the images

# Needed By (Date)
- 2/23

# Urgency
- Med

# Steps to Test

1. Add this change
2. run `npm run publish` to get webpack to rebuild the dist directories. 
3. Add a header in the WYSIWYG text field.
4. Verify you get the right facing arrow in the editor and on the page.

# Affected Projects or Products
- SOE
- Stanford Basic

# Associated Issues and/or People
- D8CORE-3516
- @katrialesser - any impact for you?

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
